pkg/vm/vm_ops_maths.go

```diff
@@ -42,6 +44,56 @@ func Sub(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
 	return NONE, nil
 }
 
+// Inc increments the stack Item's value of 1.
```

Comment:
Nit: small typo inc increments the stack item's value by one

Reply:
:+1: